sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs

```diff
@@ -1781,7 +1780,9 @@ public async Task PartitionClosingAsyncIsCalledWithOwnershipLostReasonWhenStoppi
                 })
                 .Returns<string, EventPosition, ReadEventOptions, CancellationToken>((partition, position, options, token) =>
                 {
-                    if (partition == faultedPartitionId)
+                    // Throw for the faultedPartition, but only after each partition processor has has a chance to start.
+
+                    if (partition == faultedPartitionId && partitionsBeingProcessed == partitionIds.Length)
```

Comment:
call me paranoid, but would this be safer as `>=` just in case there's a bug?  That way the test doesn't potentially hang?

Reply:
fixed and ready to merge.

Reply:
You got it!